Patches/Alpha Biomes/Items_Resource_Stuff.xml

```diff
@@ -61,93 +62,91 @@
 			</value>
 		</li>
 
-		<li Class="PatchOperationAdd">
-			<xpath>/Defs/ThingDef[defName="GU_RedWood" or defName="AB_MushroomWoodLog" or defName="AB_CrystalWood"]</xpath>
+    	<li Class="PatchOperationFindMod">
+        <mods>
+          <li>Alpha Animals</li>
+        </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+
+			<li Class="PatchOperationAdd">
+			<xpath>/Defs/ThingDef[@Name="AB_RawFishBase"]</xpath>
```

Comment:
This is the Wood Area, i think fish belongs to the area further below.

Reply:
This is included here so as to be contained within AA's conditional patch sequence, rather than need its own.